.github/workflows/very_good_cli.yaml

```diff
@@ -83,6 +85,32 @@ jobs:
       - name: Run Tests
         run: flutter pub run test ${{ matrix.test }} --run-skipped -t e2e
 
+  minDartSdk:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        test:
+          # minDartSdk tests for the test command
+          - test/src/commands/test/e2e/min_dart_sdk_test.dart
```

Comment:
We can also change the e2e test matrix to have a flutter version param. Because shouldn't our other e2e tests also work on the minimal version?

Reply:
Would you like to remove the `min_dart_sdk_test.dart` completely and make this change only in the CI?